pydoctor/extensions/deprecate.py

```diff
@@ -122,7 +122,7 @@ def deprecatedToUsefulText(ctx:model.Documentable, name:str, deprecated:ast.Call
 
     replvalue = bound_args.arguments.get('replacement')
     if replvalue is not None:
-        if astutils.node2dottedname(replvalue) is not None:
+        if isinstance(replvalue, (ast.Name, ast.Attribute)) and astutils.node2dottedname(replvalue) is not None:
```

Comment:
```suggestion
        if astutils.node2dottedname(replvalue) is not None:
```
This is probably not a required changed.